handbook/company/communications.md

```diff
@@ -59,6 +59,10 @@ Here are some topics for social media posts:
 - Industry news about device management
 - Upcoming events, interviews, and podcasts
 
+#### Promote a post on LinkedIn
```

Comment:
```suggestion
#### Fleet on LinkedIn
```
I could see this getting tangled with the responsibility on the demand page. What else could we title this?